rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -7609,6 +7609,40 @@ public void getQueuedPegoutsCount_after_RSKIP271_activation() throws IOException
         assertEquals(2, bridgeSupport.getQueuedPegoutsCount());
     }
 
+    @Test
+    public void getEstimatedFeesForNextPegOutEvent_before_RSKIP271_activation() throws IOException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(false);
+
+        BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withActivations(activations)
+                .withProvider(provider)
+                .build();
+
+        verify(provider, never()).getFeePerKb();
+        assertEquals(Coin.ZERO, bridgeSupport.getEstimatedFeesForNextPegOutEvent());
+    }
+
+    @Test
+    public void getEstimatedFeesForNextPegOutEvent_after_RSKIP271_activation() throws IOException {
```

Comment:
Please add additional tests:
- rskip is activated but there are no pegout requests, should return a 0.
- fees calculated with and without an erp Federation
Some other cases you may think of @crove-iov or @marcos-iov ?

Reply:
* what happens if fees are below dust values?
* limit case of only 1 pegout in queue should pay all fees.

Reply:
Can't think of any right now. Maybe the with/without ERP and other tests should be done for `calculatePegoutTxSize` method.

Adding a test for the release requests queue being empty should be enough to cover this method I think

Reply:
Resolved